src/Renderer.h

```diff
@@ -1,3 +1,5 @@
+#ifndef RENDERER_H
+#define RENDERER_H
```

Comment:
Is this needed?

Reply:
That's because TouchBar..h includes Renderer.h and ShaderEditor.h even though it needs neither.

Reply:
Good point. 
Fixing now.